include/openthread/link.h

```diff
@@ -137,6 +137,16 @@ OTAPI bool OTCALL otLinkIsEnergyScanInProgress(otInstance *aInstance);
 OTAPI ThreadError OTCALL otLinkSendDataRequest(otInstance *aInstance);
 
 /**
+ * This function indicates whether or not an IEEE 802.15.4 transmission is currently in progress.
```

Comment:
May want to more clearly indicate that this is a MAC-layer state.  In other words, being in the "transmit" state at the MAC layer does not actually mean the physical layer is actively transmitting.  At the same time, being in the "idle" state at the MAC layer does not mean that the physical layer is not actively transmitting (e.g. when automatically transmitting an ACK).